### PR TITLE
Implement visual changes to integrate chart

### LINKED ISSUE #676: Apply visual changes to integrate chart
[Design](https://app.zeplin.io/project/5f92a9530029eeac84ae9589/screen/5fb262fe11689064d1ab618b)

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Implement visual changes to integrate chart

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design 1](https://app.zeplin.io/project/5f92a9530029eeac84ae9589/screen/5fb262fe11689064d1ab618b), [design 2](https://app.zeplin.io/project/5f92a9530029eeac84ae9589/screen/5fb262fe8f56b77946153b31) and [design 3](https://app.zeplin.io/project/5f92a9530029eeac84ae9589/screen/5fb25e567cff9766f10ccf5e)

## Issue / Card

Fixes #676 

## Screenshots

[if relevant, include a screenshot]

### CODE REVIEW

File: static/js/src/public/details/integrate/index.js [hunk 1]
```diff
@@ -311,6 +354,19 @@ function buildChart(data) {
...
       .attr("transform", `translate(0, ${40 * i})`);
 
+    if (i === data.groups.length - 1) {
+      textGroup
+        .append("text")
+        .attr("x", chartWidth / 2)
+        .attr("y", chartHeight / 2 + 4)
+        .attr("font-family", "'Ubuntu', sans-serif")
+        .attr("font-size", "14px")
+        .attr("fill", "#666666")
```
Comment (steverydz) [inline static/js/src/public/details/integrate/index.js:364]: can this be #666 as the previous group?

File: static/js/src/public/details/integrate/index.js [hunk 2]
```diff
@@ -335,6 +391,19 @@ function buildChart(data) {
...
       .attr("transform", `translate(0, ${40 * i})`);
+
+    if (i === data.groups.length - 1) {
+      textGroup
+        .append("text")
+        .attr("x", chartWidth - iconContainerRadius * 2.5)
+        .attr("y", chartHeight / 2 + 4)
+        .attr("font-family", "'Ubuntu', sans-serif")
+        .attr("font-size", "14px")
+        .attr("fill", "#666666")
```
Comment (steverydz) [inline static/js/src/public/details/integrate/index.js:402]: same as above

**Review by steverydz — APPROVED** [review]
LGTM :+1: 

Just one minor comment in the code

**Comment by danielmutis** [comment]
@solazio one small thing: the X to close the modal seems a bit off. Can we put it in the corner?

Screenshot 2020-11-30 at 17 22 27

**Comment by solazio** [comment]
> @solazio one small thing: the X to close the modal seems a bit off. Can we put it in the corner?
> 
> Screenshot 2020-11-30 at 17 22 27

Fixed it now.

**Pull request merged**